run.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-cd `dirname $0`
+cd "$(dirname "$0")"
```

Comment:
But there is nothing wrong with the backquote form and the change reduces portability. I don't see a justification for this change.